src/services/eventbus.ts

```diff
@@ -0,0 +1,9 @@
+import mitt from 'mitt'
+
+type Events = {
+  'update-document-data': string
+  'serialize-document': undefined
+}
+
+const emitter = mitt<Events>()
+export default emitter
```

Comment:
Did we end up using the event emitter anywhere? Should we keep it in there or should we kick it out?

Reply:
No, Ill remove it